### DISCUSSION TITLE
blog/2024/03/03/kubernetes-the-hard-way/

### DISCUSSION THREAD

**Discussion by giscus** [developer-friendly/blog#45] [General]
# blog/2024/03/03/kubernetes-the-hard-way/

How to set up Kubernetes the Hard Way from scratch using Vagrant & Ansible. Learn kubernetes the hard way on your local machine without any cloud provider.

https://developer-friendly.blog/blog/2024/03/03/kubernetes-the-hard-way/

**Comment by silopolis** [comment]
Another one of your excellent posts!
Awesome work and documentation for our education and delight.
Thanks a ton for sharing.
Keep it on, please
